source/derived.tex

```diff
@@ -147,16 +147,13 @@
 a derived class and its base class subobjects can be represented by a
 directed acyclic graph (DAG) where an arrow means ``directly derived
 from''. A DAG of subobjects is often referred to as a ``subobject
-lattice''.
+lattice''. The arrows need not have a physical representation in memory.
```

Comment:
I've moved the sentence about the arrows not being physical to directly after the sentence where the arrows are first introduced. The association does indeed change, but I agree that's fine, especially with this rearrangement.